Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.UnitTests/Services/ApiGatewayRouteConfigServiceTests.cs

```diff
@@ -169,7 +171,7 @@ public void CatchAllRouteConfig()
             {
                 LambdaResourceName = "F1",
                 HttpMethod = "ANY",
-                Path = "/{proxy+}"
```

Comment:
can you add a test case to see what happens with `{proxy+}+}` (is this even valid?)

Reply:
Done